.github/workflows/azure-static-web-apps-zealous-water-04f606303.yml

```
@@ -14,8 +14,40 @@ permissions:
 
 jobs:
+  cypress_tests:
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    runs-on: ubuntu-latest
+    container:
+      image: cypress/browsers:22.15.0
+      options: --user 1001  
+    name: Cypress E2E Tests
+    env:
+      PNPM_HOME: ${{ github.workspace }}/.pnpm
+      PATH: ${{ github.workspace }}/.pnpm:${{ env.PATH }}
```

actionlint (1.7.10)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)